libcaf_net/test/endpoint_manager.cpp

```diff
@@ -103,7 +107,7 @@ class dummy_transport {
       auto& payload = x->payload;
       write_buf_.insert(write_buf_.end(), payload.begin(), payload.end());
     }
-    auto res = write(handle_, write_buf_.data(), write_buf_.size());
+    auto res = write(handle_, as_bytes(make_span(write_buf_)));
```

Comment:
Should also work without `as_bytes`.

```suggestion
    auto res = write(handle_, make_span(write_buf_));
```

Reply:
`write` expects `span<const byte>` since the implicit conversion is not possible, `as_bytes` is needed.